v2/AdminCore.Services/EventService.cs

```diff
@@ -187,6 +198,18 @@ public void IsEventValid(EventDateDto eventDates, bool modelIsHalfDay, int emplo
         throw new Exception("Holiday booked contains a half day whilst being more than one day.");
     }
 
+    public void AutoApprovePublicHolidays()
+    {
+      var publicHolidayEventTypeId = (int)EventTypes.PublicHoliday;
+      var publicHolidays = DatabaseContext.EventRepository.Get(x => x.EventTypeId == publicHolidayEventTypeId);
+      publicHolidays.Select(c =>
+      {
+        c.EventStatusId = (int)EventStatuses.Approved;
+        return c;
+      }).ToList();
+      DatabaseContext.SaveChanges();
+    }
```

Comment:
We should definitely keep this method for any batch jobs we do, but not sure why it's included now. Rather than adding events and then retroactively approving them, why don't we just set event status as approved before adding?